src/main/java/au/org/ala/biocache/service/DownloadService.java

```diff
@@ -144,6 +144,9 @@
     @Value("${download.email.template:}")
     protected String biocacheDownloadEmailTemplate;
 
+    @Value("${download.doi.ala.resolver:https://doi.ala.org.au/doi/}")
```

Comment:
Is there a commit that hasn't been sent to Github to change this to `download.doi.resolver`?

Reply:
oops, forgot to check it in.

Reply:
and please take a look at the changes in ala-install as well.